src/Chapter1.hs

```diff
@@ -632,7 +639,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n =
+    let preparedNumber = if n < 0 then n * (-1) else n
+        twoLastDigs = mod preparedNumber 100
+        lastDig = mod twoLastDigs 10
+        beforeLastDig = div twoLastDigs 10
```

Comment:
That is a wonderful solution! 👏🏼 
One hint to make your solution even shorter: you can see that you use both:

```haskell
mod ... 10
div ... 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(lastDig, beforeLastDig) = divMod twoLastDigs 10
```

You can see how we could pattern match on the pair 🙂

Reply:
Fancy function!
Thanks!